webcompat/views.py

```diff
@@ -232,7 +235,7 @@ def create_issue():
             session['form_data'] = form
             return redirect(url_for('login'))
     elif form.get('submit_type') == PROXY_REPORT:
-        response = report_issue(form, proxy=True).json()
+        response = report_issue(form, proxy=True)
```

Comment:
Can you explain this change? Is it redundant?

Reply:
oooooohhhhh. noooooo… Doh! Nice catch. I think it's because I tested by modifying report_issue so that it returns json() and then I changed my mind, but forgot to put back the `.json()`

Reply:
This is fixed and rebased.

Reply:
I think we just need to fixup the test, then we're good to merge. Thanks @karlcow.

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/home/travis/build/webcompat/webcompat.com/tests/unit/test_urls.py", line 77, in test_successful_post_new_issue
    username='yeeha'))
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/werkzeug/test.py", line 840, in post
    return self.open(*args, **kw)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/testing.py", line 200, in open
    follow_redirects=follow_redirects
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/werkzeug/test.py", line 803, in open
    response = self.run_wsgi_app(environ, buffered=buffered)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/werkzeug/test.py", line 716, in run_wsgi_app
    rv = run_wsgi_app(self.application, environ, buffered=buffered)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/werkzeug/test.py", line 923, in run_wsgi_app
    app_rv = app(environ, start_response)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/travis/build/webcompat/webcompat.com/webcompat/views.py", line 237, in create_issue
    response = report_issue(form, proxy=True).json()
AttributeError: 'dict' object has no attribute 'json'
```

Reply:
ahah now I know… why I had removed the JSON. ok let me do another round. Maybe today.